tests/superset.conf

```diff
@@ -1,18 +1,12 @@
 # -*- coding: utf-8 -*-
 # flake8: noqa
-from superset.config import *
+import os
 
 AUTH_USER_REGISTRATION_ROLE = 'alpha'
-SQLALCHEMY_DATABASE_URI = 'sqlite:///' + os.path.join(DATA_DIR, 'unittests.db')
 DEBUG = True
 SUPERSET_WEBSERVER_PORT = 8081
-
```

Comment:
This logic is now simplified given that all unit tests are run via the `tox` testing harness.